pkg/otlp/internal/serializerexporter/consumer.go

```diff
@@ -105,5 +107,19 @@ func (c *serializerConsumer) flush(s serializer.MetricSerializer) error {
 	if err := s.SendSketch(c.sketches); err != nil {
 		return err
 	}
-	return s.SendSeries(c.series)
+	flushAndSerializeInParallel := aggregator.NewFlushAndSerializeInParallel(config.Datadog)
+	iterableSeries := metrics.NewIterableSeries(func(se *metrics.Serie) {}, flushAndSerializeInParallel.BufferSize, flushAndSerializeInParallel.ChannelSize)
```

Comment:
LGTM, but it feels a bit weird to have to call an aggregator function when we are not using it on the OTLP pipeline (since OTLP metrics are aggregated in the language clients). Maybe a future opportunity for refactoring?

Reply:
👍 

Would it make sense to move to the config package? We have a number of helper structs there already.

Reply:
config (or serializer) sounds better than aggregator indeed.